src/static/js/wishlist.js

```diff
@@ -91,3 +91,45 @@ setTimeout(() => {
   document.querySelectorAll('.upForm').forEach(element => { listen(element, 'up') })
   document.querySelectorAll('.downForm').forEach(element => { listen(element, 'down') })
 }, 0)
+
+function priceToFloat(price) {
+  let cleanPrice = price
+      .replace(/[^\d.,]/g, '')  // Remove everything except digits, periods, and commas
+      .replace(/,/g, '');       // Remove commas to standardize the number
+  // If there's more than one decimal separator (.), treat the last one as the decimal point
+  if ((cleanPrice.match(/\./g) || []).length > 1) {
+      cleanPrice = cleanPrice.replace(/\.(?=.*\.)/, '');  // Remove the first period if there are multiple
+  }
+  return parseFloat(cleanPrice);
+}
+
+document.addEventListener('DOMContentLoaded', function(){
+  let sortOrderPrice = 'asc';
+  const priceArrow = document.getElementById('price-arrow');
+  const sortByPrice = document.getElementById('sort-price');
+
+  function updateArrow(column, order) {
+    if (column === 'price') {
+      priceArrow.className = order === 'asc' ? 'fas fa-arrow-up' : 'fas fa-arrow-down';
+    }
+  }
+
+  sortByPrice.addEventListener('click', function(e){
+    e.preventDefault();
+
+    const table = document.getElementById('wishlist-table');
+    const tbody = table.querySelector('tbody');
+    const rows = Array.from(tbody.querySelectorAll('tr'));
+    sortOrderPrice = sortOrderPrice === 'asc' ? 'desc' : 'asc';
```

Comment:
I would prefer to have it sort by only one direction. Clicking it again should disable sorting and return to original order. Maybe update the elements in the view to have a data-order property that you can read with `<row>.dataset.order`?

Sorting by most expensive first or least expensive first is up to you. I don't think it matters much.

Reply:
This seems like it would be different to every other table sort I've ever used?

Reply:
Here's an example of it: https://info.galaxy.casa/ships

I also wrote that so maybe I'm the weird one lol, not sure if it exists anywhere else

Reply:
That's nice but, honestly, I've never seen that before! I like that you can go back to the original table sort by clicking again but it's weird to me that I can't sort ascending order by clicking the header again.

Maybe we can get this in, get it in the wild, and see what other folks might ask for in the future?